src/snapred/ui/workflow/ReductionWorkflow.py

```diff
@@ -169,26 +173,102 @@ def _triggerReduction(self, workflowPresenter):
                 convertUnitsTo=self._reductionRequestView.convertUnitsDropdown.currentText(),
             )
 
-            response = self.request(path="reduction/", payload=request_)
-            if response.code == ResponseCode.OK:
-                record, unfocusedData = response.data.record, response.data.unfocusedData
+            # Validate reduction; if artificial normalization is needed, handle it
+            response = self.request(path="reduction/validateReduction", payload=request_)
+            if response.data:
+                self._artificialNormalizationView.updateRunNumber(runNumber)
+                response = self.request(path="reduction/grabDiffractionWorkspaceforArtificialNorm", payload=request_)
+                self._artificialNormalization(workflowPresenter, response.data, runNumber)
```

Comment:
This should probably be instead handled by a continueAnyway handler of sorts [like in NormalizationWorkflow](https://github.com/neutrons/SNAPRed/blob/c629472e8bc15ef2211a95151719a0e27ac195d8/src/snapred/ui/workflow/NormalizationWorkflow.py#L79)